package.json

```diff
@@ -91,7 +91,7 @@
     "@eslint/compat": "^1.2.3",
     "@playwright/test": "^1.48.2",
     "@podman-desktop/api": "^1.14.1",
-    "@podman-desktop/tests-playwright": "next",
+    "@podman-desktop/tests-playwright": "1.15.0-202411180939-78062c24cd",
```

Comment:
This is not correct. We should not be tied to the particular release of npm package. Next is ok. On CI we then add latest release version - aka nightly.